packages/react-mutation-mapper/src/component/oncokb/oncokbCard/BiologicalContent.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import SummaryWithRefs from '../SummaryWithRefs';
+import ReferenceList from '../ReferenceList';
+import { If, Then, Else } from 'react-if';
+import { ICache } from '../../../model/SimpleCache';
+import { Citations } from 'oncokb-ts-api-client';
+
+export const BiologicalContent: React.FunctionComponent<{
+    biologicalSummary: string;
+    mutationEffectCitations: Citations;
+    pmidData: ICache;
+}> = props => {
+    return (
+        <If
+            condition={
+                props.biologicalSummary !== undefined &&
+                props.biologicalSummary.length > 0
```

Comment:
I think we just need to do !!props.bilogicalSummary

Reply:
I think I didn't work on this component?